### ISSUE TITLE
(Linux) Lots of assertions in the main menu

### ISSUE THREAD

**Issue by DanielGibson** [rcgordon/Serious-Engine#33]
When you try to customize controls, I get an assertion from Sources/Engine/Base/Shell.cpp:704, then from line 675 then 689

it seems to be safe to click "always ignore", but this should probably still be investigated

**Comment by icculus** [comment]
This is probably my bug.

**Comment by rohit-n** [comment]
I haven't done a thorough investigation, but the first assertion seems caused by the `sys_bIFeelEnabled` symbol being undeclared. I believe it is normally declared in Engine/Engine.cpp:445 (specifically, IFeel.cpp:117), but this appears to be only done in Windows.

If iFeel support is really Windows-specific, maybe we can just set the related `mgControlsIFeelTrigger` bools to zero for a non-Windows platform at the moment?

**Comment by icculus** [comment]
Yeah, iFeel is more or less Windows-only (it doesn't have to be, but I wouldn't count on it showing up on Linux)...also, I think maybe a dead API now anyhow? I don't see anything about it on Logitech or Immersion Corp's webpages about it.
